frontend/src/components/collaborator/CollaboratorCreate.vue

```diff
@@ -51,15 +51,17 @@ export default {
   },
   data() {
     return {
-      entityProperties: ['camp', 'inviteEmail', 'role'],
+      entityProperties: ['abbr', 'camp', 'color', 'inviteEmail', 'role'],
```

Comment:
I would rather not include abbreviation and color in the create dialog.
Limit inputs fields during creation, I can still go back and edit/customize if I want to.